HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc

```diff
@@ -139,8 +141,17 @@ void HLTBTagPerformanceAnalyzer::analyze(const edm::Event& iEvent, const edm::Ev
 		}
 
 		for (auto & BtagJT: JetTag) {
+			std::map<std::string, bool> inmodule;
+			inmodule["HEP17"]=(BtagJT.first->phi() >= -0.87) && (BtagJT.first->phi() < -0.52) && (BtagJT.first->eta() > 1.3);
```

Comment:
@mtosi we need to monitor only 3 slices (HEP17, HEM17, HEP18), so any additional function will increase the length of the code. I'm not sure it is necessary in this moment. Or do you mean something else?